iotilesensorgraph/iotile/sg/sensor_log.py

```diff
@@ -46,6 +55,32 @@ def __init__(self, engine=None, model=None):
 
         self._engine = engine
         self._model = model
+        self._rollover_streaming = True
+        self._rollover_storage = True
+
+    def set_rollover(self, area, enabled):
+        """Configure whether rollover is enabled for streaming or storage streams.
+
+        Normally a SensorLog is used in ring-bufffer mode which means that old
```

Comment:
same - bufffer = buffer

Reply:
Thanks for finding these.  I just fixed both of them.  One suggestion could be to use the github `suggestion` to just put the fix inline and then I can just click accept.  I've started doing that on my own PRs as I read through the diffs to quickly fix typos

![image](https://user-images.githubusercontent.com/2424759/48149706-a14a8f00-e271-11e8-9b01-5173b55a33b0.png)

Reply:
Oh awesome -- will definitely use that!